strapi/strapi
# WYSIWYG: Fix CSS flex setup to take the toolbar into account

### What does it do?

Fixes the setup for the WYSIWYG preview pane.

### Why is it needed?

Previously it didn't take the height of the toolbar (48px) into account, which led to text being cut-off at the end. This is an alternative approach to https://github.com/strapi/strapi/pull/15560 which fixes the underlying problem instead of working around it.

### How to test it?

1. Add content to a WYSIWYG field which triggers a scroll in the preview pane
2. Verify the end of the content is not visually cut off any longer

### Related issue(s)/PR(s)

- Extends https://github.com/strapi/strapi/pull/15549
- Finally fixes https://github.com/strapi/strapi/issues/15407
- Closes https://github.com/strapi/strapi/pull/15560
